<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manager_Window</class>
 <widget class="QWidget" name="Manager_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>608</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1005</width>
    <height>608</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1005</width>
    <height>608</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Manager_Window</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/icon.png</normaloff>Resources/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#2196F5;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>541</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:25pt</string>
   </property>
   <property name="text">
    <string>Felicitariiii!!! Acum este MANAGER!!!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Logout_Button">
   <property name="geometry">
    <rect>
     <x>855</x>
     <y>10</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font-size:20pt;
border: 2px solid white;
border-radius: 9px;
background: grey;
background-color: rgba(0, 0, 0, 25%);</string>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1011</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Projects</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font-size:15pt;
border: 2px solid white;
border-radius: 9px;
background: grey;
background-color: rgba(0, 0, 0, 25%);</string>
     </property>
     <property name="text">
      <string>ADD NEW PROJECT</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>621</width>
       <height>401</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font-size:20pt;
border: 2px solid white;
border-radius: 9px;</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>91</y>
       <width>198</width>
       <height>100</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Period</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Project_ID</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Account Details</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font-size:15pt;
border-radius: 9px;
background: grey;
background-color: rgba(0, 0, 0, 25%);</string>
     </property>
     <property name="text">
      <string>First Name -&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font-size:15pt;
border-radius: 9px;
background: grey;
background-color: rgba(0, 0, 0, 25%);</string>
     </property>
     <property name="text">
      <string>Last Name -&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListView" name="viewFN">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>110</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font-size:15pt;
border-radius: 9px;
background: grey;
background-color: rgba(0, 0, 0, 25%);</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
